test/client-request.js

```diff
@@ -97,3 +99,46 @@ test('trailers', (t) => {
       })
   })
 })
+
+test('destroy socket abruptly', async (t) => {
+  t.plan(2)
+
+  const server = net.createServer((socket) => {
+    const lines = [
+      'HTTP/1.1 200 OK',
+      'Date: Sat, 09 Oct 2010 14:28:02 GMT',
+      'Connection: close',
+      '',
+      'the body'
```

Comment:
`\r\n`?

Reply:
Adding `\r\n` will show them in the end body.